doc/tutorial/web/rest/asab.webservice.md

```diff
@@ -0,0 +1,271 @@
+Creating REST HTTP webservice using  ASAB
+===
+
+## Overview
+
+On lines below, we will discuss a creation of a basic ASAB microservice application capable of 
+providing a REST interface. We will be using MongoDB as an example database we are trying to 
+perform the Create Read Update and Delete operations on.
+
+### Prerequisites
+
+1. Python version 3.6 or later
+2. Asynchronous Server App Boilerplate (ASAB) version 20.3 or later
+3. MongoDB instance
+4. Postman
+
+## Components
+
+Our sample microservice will consist of several modules, each one serving a logical purpose within the
+whole. The components are as follows (and they also hint a filestructure), and will be discussed in greater 
+detail in their respective sections below, we will go from top to bottom:   
+`myrestapi.py`, `./myrestapi/app.py`, `./myrestapi/crud/handler.py`, and `./myrestapi/crud/service.py`  
+While we start working on the microservice, we should have a testing MongoDB instance running, we will touch briefly 
+on how to quickly do that.   
+We will be using Postman to test the API and to generate json collection of available endpoints.
+
+#### myrestapi.py
+
+This is the part, from where we run our app. We begin with the shebang line, that will tell the 
+executing operating system, that we want python to execute this program.  
+Linux: `#!/usr/bin/python3`  
+Windows `#! python3`   
+OS X `#!/usr/bin/env python3`
+
+We follow this up with imports, and all we will need here is our application:   
+`from myrestapi.app import CRUDApp`   
+Next, we instantiate an object of our CRUDApp class in the __main__ of the application, and we run the app:  
```

Comment:
Once again, `CRUDApp` can be misleading `TutorialApp` is probably better, the best would be to name it based a functionality (eg. TodoApp).